pennylane/qaoa/cost.py

```diff
@@ -56,26 +178,220 @@ def maxcut(graph):
     >>> cost_h, mixer_h = qml.qaoa.maxcut(graph)
     >>> print(cost_h)
     >>> print(mixer_h)
-    (-0.5) [I0 I1] + (0.5) [Z0 Z1] + (-0.5) [I1 I2] + (0.5) [Z1 Z2]
+    (0.5) [Z0 Z1] + (0.5) [Z1 Z2] + (-1.0) [I0]
```

Comment:
Since the MaxCut Hamiltonian is now defined as a sum of other Hamiltonians, the `simplify()` method automatically sums together all like terms (in this case, the Identities).